docs/IPAM.md

```diff
@@ -0,0 +1,11 @@
+# IPAM
+
+::: pynetbox.models.ipam.Prefixes
+    handler: python
+    options:
+        members: true
+
+::: pynetbox.models.ipam.VlanGroups
+    handler: python
+    options:
+        members: true 
```

Comment:
This seems to be missing `IpAddresses`, `IpRanges`, `Aggregates`, `Vlans`, and `AsnRanges`.